src/Loading/Plugins/babylonFileLoader.ts

```diff
@@ -736,7 +736,7 @@ SceneLoader.RegisterPlugin({
                 var currentMesh: AbstractMesh;
                 for (index = 0, cache = scene.meshes.length; index < cache; index++) {
                     currentMesh = scene.meshes[index];
-                    if (currentMesh._waitingParentId !== null) {
+                    if (currentMesh._waitingParentId !== null && currentMesh._waitingParentId.length) {

```

Comment:
if it is a string, why not simply do:

```suggestion
                    if (currentMesh._waitingParentId) {
```
Will return false for empty strings

Reply:
Tested and approved!